spec/gemstash/web_spec.rb

```diff
@@ -201,6 +201,63 @@ def for(server_url, timeout = 20)
     end
   end
 
+  context "GET /api/v1/versions/:name" do
+    context "from the default upstream" do
+      let(:current_env) { Gemstash::Env.current }
+      let(:upstream) { Gemstash::Upstream.new(current_env.config[:rubygems_url]) }
+      let(:storage) { Gemstash::LocalStorage.for("gem_cache").for(upstream.host_id) }
```

Comment:
I _think_ this can become

```suggestion
      let(:storage) { Gemstash::Storage.for("gem_cache").for(upstream.host_id) }
```